rt/databinding/jaxb/src/main/java/org/apache/cxf/jaxb/FactoryClassLoader.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.jaxb;
+
+import org.apache.cxf.common.classloader.ClassLoaderUtils;
+
+public class FactoryClassLoader implements FactoryClassCreator {
+    public FactoryClassLoader() {
+    }
+    @Override
+    public Class<?> createFactory(Class<?> cls) {
+        String newClassName = cls.getName() + "Factory";
+        try {
+            return ClassLoaderUtils.loadClass(newClassName, FactoryClassLoader.class);
+        } catch (ClassNotFoundException e) {
```

Comment:
Probably we should log the exception here (debug or trace), just to help with troubleshooting